python/flow_sdk/shim_airbyte_cdk.py

```diff
@@ -108,6 +153,7 @@ def discover(self, discover: request.Discover) -> response.Discovered:
 
             config: StreamResourceConfig = {
                 "stream": stream.name,
+                "namespace": stream.namespace,
```

Comment:
We were already setting this just below, if it evaluated as truthy. Do we _need_ to always explicitly set it?

[Resource path pointers resolution treats undefined and null identically.](https://github.com/estuary/flow/blob/f4c93998569fa3681b99144b4880ce663ef609c9/crates/agent/src/discovers/specs.rs#L67-L87)

Reply:
Oop, tunnel vision strikes again. Removed 👍